samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala

```diff
@@ -55,8 +52,10 @@ class KafkaCheckpointManager(
                               socketTimeout: Int,
                               bufferSize: Int,
                               fetchSize: Int,
+                              val getSystemConsumer: () => SystemConsumer,
```

Comment:
Don't need the 'val' in params.

Reply:
I thought it was needed if we want this to be a member of the class. Looks like it is not required.